### PR TITLE
RE: Update Primary Hero loading skeleton to reflect a hero with an image

### LINKED ISSUE #9049: Update Primary Hero loading skeleton to reflect a hero with an image
This is a follow-up to https://github.com/mozilla/addons-frontend/pull/9045.

### Describe the problem and steps to reproduce it:

Visit any page on the site, and then visit the home page.

### What happened?

You see a loading skeleton, but it doesn't match the layout of the eventual primary hero element.

### What did you expect to happen?

The loading skeleton should more closely match the layout of the primary hero element with an image.

### Anything else we should know?

When the loading skeleton for the primary hero element was first implemented, it was believed that, for the most part, the primary hero *would not* contain an image. That assumption has been reversed and now the primary hero will almost always contain an image, so we should adjust the loading skeleton accordingly.

### PR DESCRIPTION
**Pull request by xlisachan**
Fixes #9049 

This pull request updates the primary hero loading skeleton to reflect a hero with an image.

https://user-images.githubusercontent.com/28129806/131353745-19785f98-07c2-4fb4-9292-9c25cbd547cc.mov

Please review when you have a moment. Thank you!

### CODE REVIEW

**Review by bobsilverberg — COMMENTED** [review]
This looks good, thanks @xlisachan. I think it might look better if the loading indicator for the image was a box, instead of just a line. I think you can achieve that by altering the markup and css a bit. Do you want to give that a try?

**Comment by xlisachan** [comment]
Thanks for the feedback, @bobsilverberg ! The latest push reflects an updated image loading indicator to resemble a box instead of a line.

https://user-images.githubusercontent.com/28129806/132233006-8ae170c3-9429-4360-b005-aa4e15da1611.mov

Please review when you have a moment. Thank you!

File: src/amo/components/HeroRecommendation/styles.scss [hunk 1]
```diff
@@ -41,6 +41,16 @@
   }
 }
 
+.HeroRecommendation-image-loading {
+  @include respond-to(medium) {
+    width: 40%;
+  }
+
+  .LoadingText {
+    height: 14em;
```
Comment (bobsilverberg) [inline src/amo/components/HeroRecommendation/styles.scss:50]: This looks good, but I wonder where the value of `14em` comes from? What happens if you use `100%` instead?

Comment (xlisachan) [reply src/amo/components/HeroRecommendation/styles.scss]: `100%` works! `em` was the unit used elsewhere for the height and I was trying to stay consistent. (`14em` is the approximate height of the image when the screen is at 1024px width). I have pushed through an updated stylesheet.

https://user-images.githubusercontent.com/28129806/132370694-f656d717-08d3-41f3-8852-0f837c084e4f.mov

**Review by bobsilverberg — APPROVED** [review]
Looks great, thanks for taking this on @xlisachan!

**Pull request merged**